### PR TITLE
Make subject optional

### PR DESCRIPTION
**Pull request by avtakkar**

### CODE REVIEW

File: specs-go/v1/manifest.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type Manifest struct {
 
 	// Subject is an optional reference to any existing manifest within the repository.
 	// When specified, the artifact is said to be dependent upon the referenced subject.
-	Subject Descriptor `json:"subject"`
+	Subject Descriptor `json:"subject,omitempty"`
```
Comment (shizhMSFT) [inline specs-go/v1/manifest.go]: How about this?
```suggestion
	Subject *Descriptor `json:"subject,omitempty"`
```

Comment (avtakkar) [reply specs-go/v1/manifest.go]: Thanks, updated

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: specs-go/v1/manifest.go (hunk 1 above)
Comment (northtyphoon) [reply specs-go/v1/manifest.go]: Do you consider to update other optional string properties in the spec?

**Review by michaelb990 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**